docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -390,7 +390,9 @@ Methods
 | ``sort_index``              | P                               |                                  | ``N`` if given the ``key`` param. ``N`` if         |
 |                             |                                 |                                  | ``axis == 1``, or MultiIndex.                      |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``sort_values``             | P                               |                                  | ``N`` if given the ``key`` param or ``axis == 1``  |
+| ``sort_values``             | P                               | ``key``                          | ``N`` if given the ``key`` param or ``axis == 1``. |
+|                             |                                 |                                  | Snowpark pandas currently uses stable sort when    |
```

Comment:
Some corrections:

- "when sorting the index values" doesn't seem to apply since this is about `DataFrame`. Anyway, I would delete this fragment because it's redundant.
- This description makes it sound like "stable sort" is an algorithm, but it's a description of an algorithm.
- We should mention that Snowpark pandas ignores the `kind` parameter completely. We currently warn `choice of sort algorithm 'quicksort' is ignored. sort kind must be 'stable' or None`.

So I suggest something like 

`The ``kind`` paramater has no effect. Snowpark pandas always uses a stable sort algorithm, while pandas by default does not.`